build-tools/attributions-generator.sh

```diff
@@ -10,10 +10,15 @@ set -x
 ATTR_GEN_IMG=f5networksdevel/attributions-generator:latest
 docker pull ${ATTR_GEN_IMG}
 
+LOCAL_USER_ID=$(id -u)
+if [ "$GITLAB_CI" == true ]; then
+  LOCAL_USER_ID=9001
```

Comment:
Why are we hardcoding this?

Reply:
As GITLAB container does not have any other user created we are using the root user, however we need some id for CIS container to run, hence using 9001, earlier it was using the ID of user which is running the make prod command.